### PR TITLE
Fix for AMD require import of Dropzone

### PR DESCRIPTION
**Pull request by a904guy**
Fixes AMD require import of Dropzone.

### CODE REVIEW

**Comment by a904guy** [comment]
@vvakame Another AMD fix.
@nvivo Can approve, it is a simple change.

**Comment by a904guy** [comment]
I shouldn't of tagged the merge to master, I've got a PNotify fix as well I want to merge as well.

File: dropzone/dropzone.d.ts [hunk 1]
```diff
@@ -1,95 +1,128 @@
...
+	emit( eventName: "dragover", e: Event );
+	emit( eventName: "dragleave", e: Event );
 }
 
 interface JQuery {
-    dropzone(options: DropzoneOptions): Dropzone;
+	dropzone( options: DropzoneOptions ): Dropzone;
+}
+
+declare module "Dropzone" {
```
Comment (vvakame) [inline dropzone/dropzone.d.ts]: is this right? not `declare module "dropzone"` ?

**Comment by vvakame** [comment]
@nvivo could you review this PR?

**Comment by nvivo** [comment]
@a904guy makes sense, it's a simple addition..

2 small things for the sake of best practices:

1. Change the module export to lowercase to match the file name as @vvakame proposed;
2. Try commiting only the changes you want to make - the diff shows 112 additions and 79 deletions and you only added 1 line.

**Comment by a904guy** [comment]
@nvivo Actually I added a bunch, but I also reformatted the code to tabs (.editorconfig got me, hence the big diff).

New code is mixed in between these lines. I brought it up to date with the 4.0.1 release.
https://github.com/a904guy/DefinitelyTyped/blob/master/dropzone/dropzone.d.ts#L74-L103

**Comment by a904guy** [comment]
@vvakame :+1: ?

**Comment by nvivo** [comment]
Fine for me.

**Comment by a904guy** [comment]
Sweet

File: dropzone/dropzone.d.ts (hunk 1 above)
Comment (a904guy) [inline dropzone/dropzone.d.ts]: Technically per their documentation and source, they reference their objects with capital D. 

http://www.dropzonejs.com/#create-dropzones-programmatically

**Comment by vvakame** [comment]
@nvivo thanks!  
@a904guy thanks mate!

**Comment by a904guy** [comment]
@vvakame aye, thank you!

**Pull request merged**